requirements.txt

```diff
@@ -1,7 +1,9 @@
-asgiref==3.3.1
+asgiref==3.3.4
 coverage==5.5
-Django==3.1.7
+Django==3.2
 psycopg2==2.8.6
+python-memcached==1.59
```

Comment:
No need to install memcached in #26, but ok 😉

Reply:
does it need only to debug?

Reply:
How it's connected to debugging?

Reply:
in this case why did we install it?

Reply:
i didn't fix it in my instructions

Reply:
The main idea of #26 is to update Django version from 3.1 to 3.2
And you added python-memcached which only will be necessary in #28, so that's the reason why I payed your attention to than.

Reply:
Got it👍